Dockerfile

```diff
@@ -9,5 +9,4 @@ WORKDIR /model
 
 ENV PYTHONPATH "${PYTHONPATH}:/model"
 
-ENTRYPOINT ["gunicorn"]
-CMD ["-w", "4", "-b", "0.0.0.0:8000", "-t", "150", "-k", "aiohttp.worker.GunicornWebWorker", "model.app:get_app()"]
+CMD ["gunicorn", "-w", "4", "-b", "0.0.0.0:8000", "-t", "150", "-k", "aiohttp.worker.GunicornWebWorker", "model.app:get_app()"]
```

Comment:
Nice, `ENTRYPOINT` is kinda nasty since you can execute any other command which is terrible for debugging a container.